README.md

```diff
@@ -6,15 +6,17 @@ You can install orloj through the R command line. Please make sure to run R in a
 
 1. Install the required Bioconductor packages:
 ```
-source("https://bioconductor.org/biocLite.R")
-biocLite("flowCore")
-biocLite("FlowSOM")
-biocLite("edgeR")
+source("https://bioconductor.org/packages/3.11/bioc/")
```

Comment:
I think this line is not necessary, and in fact could be a security risk. It tries to run an HTML as R code. Could you please remove it? Thanks!